.github/workflows/Containerfile

```diff
@@ -49,8 +49,6 @@ RUN apt install -y --no-install-recommends \
     python3-dbus
 
 # Install doc dependencies
-RUN apt install -y --no-install-recommends \
-    python3-sphinx-copybutton \
-    python3-sphinxext-opengraph \
-    furo \
-    python3-sphinx
+RUN apt install -y --no-install-recommends python3-pip
+RUN pip install --user --break-system-packages furo">=2024.04.27" \
```

Comment:
Is there a reason to install this into user instead of system? If it's installed to system we wouldn't need to adjust the PATHs in the workflow.

Reply:
I generally wanted to avoid conflicts/overlaps. If one of the Python modules gets installed via `apt` first through some dependency, `pip` will fail to install it in system.

Reply:
Mh, isn't that what `--ignore-installed` is for?

Reply:
`--ignore-installed` is a bit slippery-slope, because it overwrites the system version with the pip's version. 

If `apt` installed `foobar 2.2.0`, the something might depend on that version and might break with pip which always fetches the latest right now under the current config.

Reply:
which is also why I set these env vars here so that it doesn't pollute anything else. Without the env var, the packages won't be found.

Reply:
Alright, let's go with the current version then.